src/args.rs

```diff
@@ -731,8 +731,10 @@ impl<'a> ArgMatches<'a> {
         let mut specs = vec![
             "path:fg:magenta".parse().unwrap(),
             "line:fg:green".parse().unwrap(),
+            "column:fg:yellow".parse().unwrap(),
```

Comment:
Right, but if you read #377, then I'm specifically calling into question why humans are reading column numbers, especially when the match itself is colored.